docs/configuration/recording_rules.md

```diff
@@ -86,6 +86,9 @@ name: <string>
 # rule can produce. 0 is no limit.
 [ limit: <int> | default = 0 ]
 
+# Delay rule evaluation of this particular group by the specified duration.
+[ evaluation_delay: <duration> | default = rules.evaluation_delay ]
```

Comment:
If I'm not mistaken, this would introduce the first config item that overrides a command-line flag... at least the other per-rule-group settings only override settings from the global configuration file. Makes me wonder:

* Do we need this per group at all (at least initially, is someone asking for it already?)?
* If so, would it be better to make this a global config option rather than a flag, to be more in line with what we do already?

Reply:
> Do we need this per group at all (at least initially, is someone asking for it already?)?

From experience - this has proven significantly useful. 

The argument was that we want to have a lower evaluation delay for all of metrics to safeguard for remote write delays but we want an even bigger one (or no delay due to its time sensitivity) for our "system rules" as those monitor 100s of 1000s of things were fast positives trigger an alert storm. With the same argument being in reverse, we don't want any delay for all of our metrics but we want this particular group to have a delay because where the remote write source where they come from tends to lag behind sometimes due to its size.

> If so, would it be better to make this a global config option rather than a flag, to be more in line with what we do already?

I can see a world where you might want to control the flag at a server level - perhaps what I should introduce is a guard to make sure the config option cannot be lower than the server flag which I think it's what makes sense in my head.